### ISSUE TITLE
Luminosity contrast ratio of keyboard focus indicator on Authorize button fails to meet the required ratio of 3:1: A11y_VS Code extension for API Center_Generate HTTP file_Authorize_Non Text Contrast.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#105]
### GitHub Tags
#A11yTCS; #A11ySev2; #Mac; #WCAG1.4.11; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #A11yMAS; #Non Text Contrast; #A11YWCAG2.1; #device;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:
Create an API center in azure portal.
Open Visual studio Code.
Tab till API center extension in the left side panel and press enter.
Tab till definition under the Api name which you have created, and press enter.
The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
Go to "Generate HTTP file" using the down arrow and press enter.
An HTTP file will appear.
Tab till Authorize button.
Verify whether the Luminosity contrast ratio of keyboard focus indicator on Authorize button meet the required ratio of 3:1 or not.
### Actual Result:
Keyboard focus is going to the background control when navigating to the "Available authorizations" Page.
**Note: This issue is observed through out the feature.**

### Expected Result:
Luminosity contrast ratio of visual focus indicator on share button should be greater than or equal to required ratio of 3:1.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors

### Attachment:

**Comment by AnKushSingh05** [comment]
@formulahendry Can you please provide us the standalone sample app or Code pen link to route this bug to external team.

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222939

**Comment by Yash14j** [comment]
### Github Tags:
#External;#External:SwaggerUI;#eDAD:3p:222939;ThirdParty;

**Comment by formulahendry** [comment]
vs code bug: https://github.com/microsoft/vscode/issues/237882

**Comment by AnKushSingh05** [comment]
Adding the tag #ResolvedByTT;
